site/content/docs/5.2/utilities/z-index.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: docs
+title: z-index
+description: Use our low-level `z-index` utilities to quickly change the stack level of an element or component.
+group: utilities
+toc: true
+---
+
+## Example
+
+{{< added-in "5.3.0" >}}
+
+Use `z-index` utilities to stack elements on top of one another. Requires a `position` value other than `static`, which can be set with custom styles or using our [position utilities]({{< docsref "/utilities/position/" >}}).
+
+{{< callout >}}
+We call these "low-level" `z-index` utilities because of their default values of `-1` through `3`, which we use for the layout of overlapping components. High-level `z-index` values are used for overlay components like modals and tooltips.
+{{< /callout >}}
+
+{{< example class="bd-example-zindex-levels position-relative" >}}
```

Comment:
I think my 🧠 doesn't process the semi-transparent squares correctly 😄 
Would it be possible to rather use plain squares with different colors or stay with the blue squares (or gray squares to be less violent visually) but with a border to distinguish the different layers?